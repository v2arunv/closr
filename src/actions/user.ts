import {IUser} from "../models/users";

export interface IUserAction {
    type: string,
    user: IUser,
}

export const setUser = (user: IUser): IUserAction => {
    return <IUserAction> {
        type: 'SET_USER',
        user: user,
    }
};