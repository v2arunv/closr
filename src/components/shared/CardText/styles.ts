import {StyleSheet} from "react-native";
import common from "../../../common/styles";

const styles = StyleSheet.create({
    container: {
        flex: 1,
        padding: 10,
        backgroundColor: '#FFFFFF',
        borderBottomColor: common.borderColor,
        borderBottomWidth: 1,
    }

});

export default styles;