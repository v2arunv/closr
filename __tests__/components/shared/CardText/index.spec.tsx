import React from 'react';
import CardText from '@sharedComponents/CardText';
import renderer from 'react-test-renderer';
import user from '@mocks/users';

test('renders correctly', () => {
    const tree = renderer.create(
        <CardText>
        </CardText>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});