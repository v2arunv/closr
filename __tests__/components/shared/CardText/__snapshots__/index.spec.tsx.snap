// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderBottomColor": "#dfdfdf",
      "borderBottomWidth": 1,
      "flex": 1,
      "padding": 10,
    }
  }
>
  <Text />
</View>
`;
